pkg/cmd/roachtest/tests/admission_control_latency.go

```diff
@@ -371,7 +369,13 @@ type backfill struct{}
 var _ perturbation = backfill{}
 
 func (b backfill) setupMetamorphic(rng *rand.Rand) variations {
-	return newMetamorphic(b, rng)
+	v := newMetamorphic(b, rng)
+	// TODO(#133114): The backfill test can cause OOM with low memory
+	// configurations.
+	if v.mem == spec.Low {
+		v.mem = spec.Standard
```

Comment:
Before the change, `Standard` and `High` were chosen with p=1/2. Now, `Standard` is chosen with p=2/3. Perhaps it's fine?

Reply:
Yes - this is probably OK for now. Once we fix the low memory config for this test we will re-enable.